keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -582,11 +636,12 @@ and creating new ones, as well as about cluster-level services (e.g. DNS):
       - Impact of its degraded performance or high-error rates on the feature:
 -->
 
+No.
+
 ### Scalability
 
 ###### Will enabling / using this feature result in any new API calls?
 
-
 This feature does not add any new API calls. 
```

Comment:
The event emission requires an event write every for every failure, correct?  Writes during failure modes are ok, but we should know they're happening.